src/Aspire.Dashboard/Components/ResourcesGridColumns/LogLevelColumnDisplay.razor

```diff
@@ -0,0 +1,52 @@
+﻿@namespace Aspire.Dashboard.Components
+
+@using Aspire.Dashboard.Otlp.Model
+@using Aspire.Dashboard.Resources
+@using Microsoft.Extensions.Logging
+
+@inject IJSRuntime JS
+@inject IStringLocalizer<ControlsStrings> ControlsStringsLoc
+
+@{
+    var id = Guid.NewGuid().ToString();
+}
+
+@if (LogEntry.Severity is LogLevel.Error or LogLevel.Critical)
+{
+    <FluentIcon Id="@id" Icon="Icons.Filled.Size16.ErrorCircle" Color="Color.Error" Class="severity-icon"/>
+}
+else if (LogEntry.Severity is LogLevel.Warning)
+{
+    <FluentIcon Id="@id" Icon="Icons.Filled.Size16.Warning" Color="Color.Warning" Class="severity-icon"/>
+}
+
+@if (TryGetErrorInformation(out var errorInfo))
```

Comment:
This came up before: exceptions aren't tied to the severity. It's possible to have an exception at any level. The icon shouldn't be used for this.

Also, how would anyone know to hover the cursor over the icon to get exception detail?

Reply:
What about adding an icon to indicate the log entry has exception data at the end of the message? Where the red box is:

![image](https://github.com/dotnet/aspire/assets/303201/e800724b-2033-4044-a32b-f6676529bf54)

Hovering over that shows the error detail popup.

Also, while we're here, there should be a copy button for the message, like there is for the Source column on the resources page.